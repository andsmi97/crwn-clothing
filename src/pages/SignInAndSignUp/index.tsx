import React from "react";

import "./styles.scss";
import SignIn from "../../components/SignIn";

export default (): JSX.Element => {
  return (
    <div className="sing-in-and-sign-up">
      <SignIn />
    </div>
  );
};
