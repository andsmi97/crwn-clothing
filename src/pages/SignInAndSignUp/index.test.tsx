import React from "react";
import { shallow } from "enzyme";
import SignInAndSignUp from "../SignInAndSignUp";

describe("<SignInAndSignUp />", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<SignInAndSignUp />);
    expect(wrapper).toMatchSnapshot();
  });
});
