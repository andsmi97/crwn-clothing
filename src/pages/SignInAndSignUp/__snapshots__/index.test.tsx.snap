// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignInAndSignUp /> matches snapshot 1`] = `ShallowWrapper {}`;
